scala-exercises-server/app/assets/stylesheets/path/css-directory

```diff
@@ -0,0 +1,23 @@
+/*
+Errno::ENOENT: No such file or directory @ rb_sysopen - path/sass-directory
+
+Backtrace:
+/Users/israelperezglez/.rvm/gems/ruby-2.1.1/gems/sass-3.4.16/lib/sass/plugin/compiler.rb:482:in `read'
```

Comment:
I'm guessing all of these files pointing to @israelperezglez mahcine shouldn't be checked in and perhaps where not added to .gitignore?